cloud/administration/resource_access.md

```diff
@@ -0,0 +1,43 @@
+---
+id: ram
+title: Granting users rights on resources
+---
+
+Resource access rules allow you to grant users the right to see specific [resources](../resources/glossary.md#resource).
+
+By default, all users can see all resources. This is because a default access rule called "ALL" is created automatically when the platform is installed.
+
+Administrators can create more specific rules to determine which users can see which resources. Only administrators can access the **Resource Acces Management** screen.
+
+You can create as many rules as you want. If a user is mentioned in two access rules with different rights, the more permissive rule will take precedence.
+
```

Comment:
The more (most?) permissive rule...
Sûr? je vois pas trop ce que ça veut dire. Sauf (grosse) erreur de ma part, c'est surtout cumulatif.

Reply:
t'as raison ça prête à confusion